modules/sc-mesh-secure-deployment/src/nats/src/validation.py

```diff
@@ -297,3 +298,18 @@ def validate_mptcp(mptcp: str) -> bool:
         return False
     except (ValueError, TypeError, AttributeError):
         return False
+
+def validate_slaac(slaac: str) -> bool:
+    """
+    Validates a given slaac ifaces.
+    Returns True if the slaac ifaces are valid, False otherwise.
+    """
+    try:
+        for slaac_if in slaac.split():
+            if (not slaac_if.startswith("wl") and
+                    not slaac_if.startswith("halow") and
+                    not slaac_if.startswith("usb")):
+                return False
```

Comment:
Thank you for checking but I think this might create problems: for example we might want SLAAC on an RJ45 interface so a name like eth0, en[psx]* would get a False 🙂 And although this happens much more often in BSD than in Linux, I wonder if we might get yet another interface prefix just because some driver decided to call the interface blah0 or something 🤔

Reply:
Your suggestion is to remove validation for SLAAC parameter?

Reply:
I mean "this validation" of checking if the interface exists before launching the daemon with a typoed config (which might then be restarted many times until e.g. systemd realizes this is restarting too many times), and I just realized that the startsWith('wl') wouldn't catch the "wlna0" typo 🙂

Reply:
This is more like typo-catcher and needs improvements.